src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -434,6 +443,7 @@ def _filter_privacy_request_queryset(
     external_id: Optional[str] = None,
     action_type: Optional[ActionType] = None,
     include_consent_webhook_requests: Optional[bool] = False,
+    include_deleted_requests: Optional[bool] = False,
```

Comment:
same here, by default we will filter out deleted requests, unless explicitly included